### PR TITLE
Revert "CAKE-5199 | Tweak Ad insert logic"

### PR DESCRIPTION
**Pull request by cschucode**
This reverts commit 3d7e3b3dc239db4c98a7450e024f4c64e09978bf.

## Links

* http://wikia-inc.atlassian.net/browse/CAKE-5256
* Reverts https://wikia-inc.atlassian.net/browse/CAKE-5199

## Description

- Reverts the change made to tweak ad insert logic
- Removed addition of `affiliate-slot` class

## Reviewers

@Wikia/cake

### CODE REVIEW

**Comment by JoshuaRogan** [comment]
Let's make sure we test this on a sandbox but LGTM (hyper-conservative on this one)

**Review by vforge — DISMISSED** [review]

**Comment by vforge** [comment]
Ans let's as AdEng how we can run tests on sandbox for this one

**Comment by cschucode** [comment]
@emadry ran the ad-eng tests on sandbox-s1 and gave us the green light.

**Review by JoshuaRogan — APPROVED** [review]

File: app/components/article-content.js [hunk 1]
```diff
@@ -457,7 +457,7 @@ export default Component.extend(
             element.classList.add(shortClass);
             element.insertAdjacentHTML('beforeend',
               '<tr class=infobox-expand><td colspan=2><svg viewBox="0 0 12 7" class="icon">'
-       + '<use xlink:href="#chevron"></use></svg></td></tr>');
+              + '<use xlink:href="#chevron"></use></svg></td></tr>');
```
Comment (JoshuaRogan) [inline app/components/article-content.js]: What is this change ?

**Comment by JoshuaRogan** [comment]
https://github.com/Wikia/mobile-wiki/pull/1851/commits/c1bb2ddebeab8c713119cf0522184d0f4adb082b introduces some weird format changes

File: app/components/article-content.js (hunk 1 above)
Comment (cschucode) [reply app/components/article-content.js]: Looks like my VSCode added that extra space.

**Pull request merged**